app/controllers/stories/tagged_articles_controller.rb

```diff
@@ -44,9 +44,7 @@ def set_number_of_articles
     end
 
     def set_stories
-      @stories = Articles::Feeds::LargeForemExperimental
-        .new(number_of_articles: @number_of_articles, tag: @tag, page: @page)
-        .published_articles_by_tag
+      @stories = Articles::Feeds::Tag.call(@tag, number_of_articles: @number_of_articles, page: @page)
```

Comment:
This seems much clearer to me, as there was no direct relation between the tag feed and any particular home feed strategy.